### ISSUE TITLE
Hello, we've noticed an issue with your iOS app. Please delete this app and re-install it from the App Store.

### ISSUE THREAD

**Issue by marsrobertson** [stephenou/fruitionsite#199]
Duplicate of #55

Also from Google: https://twitter.com/jassification/status/1335929405185556480

I use Notion for research / wiki. I use embedded tweet functionality all the time for references.

**Comment by PersistVenture** [comment]
bump

**Comment by aayush-krishnan** [comment]
Hi I'm facing the same issue. Were you able to resolve it?

**Comment by christen-wtf** [comment]
Bump. This hasn't always happened to me. Not sure how to replicate it.

**Comment by jayviddy** [comment]
Yep, happening to me all of a sudden this week with no changes to the fruition site.

**Comment by kms0219kms** [comment]
bump

**Comment by kms0219kms** [comment]
duplicate #242. How about remove MetaRewriter Class?

**Comment by fobin** [comment]
Same issue here. Happens on Desktop Safari but not on Desktop Chrome.

**Comment by kms0219kms** [comment]
> Same issue here. Happens on Desktop Safari but not on Desktop Chrome. 

@fobin Can you remove MetaRewriter ?

**Comment by fobin** [comment]
> > Same issue here. Happens on Desktop Safari but not on Desktop Chrome.
> 
> @fobin Can you remove MetaRewriter ?

I tried it but it didn't help me. I did then changes in this PR and it's now fixed: https://github.com/stephenou/fruitionsite/pull/243/files/b980cd435f420f218fb4780393455eb9710e508f

**Comment by marsrobertson** [comment]
I clicked around, tried various things, I think it is working now: https://wiki.basex.com

**PRO TIP:** use private mode in Safary.

Then to verify it work for real, I had to clear cache:
<img width="699" alt="image" src="https://github.com/stephenou/fruitionsite/assets/44369284/daec95e6-2aa6-4419-8461-a2ba7f8de018">

**Comment by JJCoderGo** [comment]
Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.

**Comment by luanaboucher** [comment]
> Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.

same here...

**Comment by fobin** [comment]
> > Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.
> 
> same here...

And have you tried what I said in this comment: https://github.com/stephenou/fruitionsite/issues/199#issuecomment-1709688358

**Comment by luanaboucher** [comment]
> > > Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.
> > 
> > 
> > same here...
> 
> And have you tried what I said in this comment: [#199 (comment)](https://github.com/stephenou/fruitionsite/issues/199#issuecomment-1709688358)

I did...

**Comment by fobin** [comment]
> > > > Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.
> 
> > > 
> 
> > > 
> 
> > > same here...
> 
> > 
> 
> > And have you tried what I said in this comment: [#199 (comment)](https://github.com/stephenou/fruitionsite/issues/199#issuecomment-1709688358)
> 
> 
> 
> I did... 

And you did test this afterwards in Safari with Private Browsing or cleared all caches? It has to be sure that it doesn't load cached responses.

**Comment by luanaboucher** [comment]
> > > > > Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > same here...
> > 
> > 
> > > 
> > 
> > 
> > > And have you tried what I said in this comment: [#199 (comment)](https://github.com/stephenou/fruitionsite/issues/199#issuecomment-1709688358)
> > 
> > 
> > I did...
> 
> And you did test this afterwards in Safari with Private Browsing or cleared all caches? It has to be sure that it doesn't load cached responses.

It's fixed :D 
Just after I did the change in the worker I tried with Private Browsing Safari and it didn't work. I guess I had to wait a little ehehe
Thank you so much!!!

**Comment by JJCoderGo** [comment]
> > > > > > Any luck? Both my iPhone and MacBook Pro can see this message whenever I try to open website hosted via Notion site.
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > same here...
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > And have you tried what I said in this comment: [#199 (comment)](https://github.com/stephenou/fruitionsite/issues/199#issuecomment-1709688358)
> > > 
> > > 
> > > I did...
> > 
> > 
> > And you did test this afterwards in Safari with Private Browsing or cleared all caches? It has to be sure that it doesn't load cached responses.
> 
> It's fixed :D Just after I did the change in the worker I tried with Private Browsing Safari and it didn't work. I guess I had to wait a little ehehe Thank you so much!!!

My still did not work, even I cleared the caches, cookies and history, closed Safari and opened in Private mode...Should I change anything for the worker?

**Comment by grrowl** [comment]
> > > Same issue here. Happens on Desktop Safari but not on Desktop Chrome.
> > 
> > 
> > @fobin Can you remove MetaRewriter ?
> 
> I tried it but it didn't help me. I did then changes in this PR and it's now fixed: https://github.com/stephenou/fruitionsite/pull/243/files/b980cd435f420f218fb4780393455eb9710e508f

this worked for me

**Comment by JJCoderGo** [comment]
> > > > Same issue here. Happens on Desktop Safari but not on Desktop Chrome.
> > > 
> > > 
> > > @fobin Can you remove MetaRewriter ?
> > 
> > 
> > I tried it but it didn't help me. I did then changes in this PR and it's now fixed: https://github.com/stephenou/fruitionsite/pull/243/files/b980cd435f420f218fb4780393455eb9710e508f
> 
> this worked for me

Do you mind showing me how to do it step by step...hope it will not give you any trouble, as I am a zero-knowledge website designer/builder. Much appreciate it!

**Comment by JJCoderGo** [comment]
I tried to update the worker by using the code from the link above, but I am getting a message from CloudFlare "Uncaught SyntaxError: Unexpected token ')'  at worker.js:541:2", am I doing anything wrong?

**Comment by JJCoderGo** [comment]
I just updated the worker. I will wait and see if it can work out.

**Comment by JJCoderGo** [comment]
Problem solved by simply updating the worker. Thanks, guys.

**Comment by invictivi** [comment]
This solution could be listed in the FAQ :) helped me 🙇

**Comment by tranthaihoang** [comment]
Solution: https://github.com/stephenou/fruitionsite/issues/55#issuecomment-1978266460
